src/questions/dai.yaml

```diff
@@ -0,0 +1,149 @@
+- uuid: eadea988-9447-41a8-b071-199750b81925
+  type: SingleChoice
+  content: Who created DAI?
+  hint: Definitely not the US government
+  explanation: DAI was created by the MakerDAO project, which was launched in 2017 by Rune Christensen, the CEO of MakerDAO.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: The US government
+      key: A
+    - content: MakerDAO
+      key: B
+    - content: Satoshi Nakamoto
+      key: C
+    - content: Vitalik Buterin
+      key: D
+- uuid: 28156f57-3733-4934-8199-5c65a26f822d
+  type: SingleChoice
+  content: What is DAI primarily used for?
+  hint: NoHint
+  explanation: DAI is used in various ways, including as a medium of exchange, a store of value, and a collateral asset. It has gained widespread adoption in the cryptocurrency community and has been used in a number of decentralized finance (DeFi) applications.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: A medium of exchange
+      key: A
+    - content: A store of value
+      key: B
+    - content: A collateral asset
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 6115919e-358e-4d2c-936e-d50f08f384b3
+  type: SingleChoice
+  content: Is MakerDAO and the DAI stablecoin open-source?
+  hint: NoHint
+  explanation: MakerDAO is an open-source project that is maintained by a team of developers and a community of volunteers.
+  answerKeys:
+    - A
+  difficultyLevel: Low
+  choices:
+    - content: Yes
+      key: A
+    - content: No
+      key: B
+- uuid: 6bacd29b-b431-48a6-82fb-50f31becf94f
+  type: SingleChoice
+  content: What did the security audit of the Dai Token (DAI) smart contract conclude about the smart contract's safety for deployment?
```

Comment:
I guess there had been many audits.  Typically every release is audited and MakerDAO had a lot of code releases